tests/blacken_docs_test.py

```diff
@@ -672,3 +672,27 @@ def test_format_src_rst_pycon_preserves_output_indentation():
     )
     after, _ = blacken_docs.format_str(before, BLACK_MODE)
     assert after == before
+
+
+def test_format_src_rst_pycon_no_prompt():
+    before = (
+        '.. code-block:: pycon\n'
+        '\n'
+        '    pass\n'
```

Comment:
I think this should produce an error unless `--skip-errors` is set

Reply:
Consider the following code block:

```rst
.. code-block:: pycon

    # Comment about next line
    >>> pass
```

Technically, this is not output one would ever see in a Python console as they aren't able to add comments above a prompt. However, even though this is wrong, I'm not sure it is helpful to present an error for the user. This shares the property that the first non-empty line is not prefixed with `>>> `, which I believe is the state you're suggesting should error. WDYT?

Reply:
hmmm yeah `doctest` accepts this so it's probably ok?


```console
$ python3 -m doctest t.rst -vv
Trying:
    pass
Expecting nothing
ok
1 items passed all tests:
   1 tests in t.rst
1 tests in 1 items.
1 passed and 0 failed.
Test passed.
```